authV2_json.go

```diff
@@ -69,22 +69,25 @@ func (s *jsonSignature) MarshalJSON() ([]byte, error) {
 }
 
 type jsonInputs struct {
-	GenesisID            *core.ID          `json:"genesisID"`
-	ProfileNonce         *jsonInt          `json:"profileNonce"`
-	AuthClaim            *core.Claim       `json:"authClaim"`
-	AuthClaimIncProof    *merkletree.Proof `json:"authClaimIncMtp"`
-	AuthClaimNonRevProof *merkletree.Proof `json:"authClaimNonRevMtp"`
-	TreeState            TreeState         `json:"treeState"` // Identity state
-	GISTProof            GISTProof         `json:"gistProof"`
-	Signature            *jsonSignature    `json:"signature"`
-	Challenge            *jsonInt          `json:"challenge"`
+	GenesisID          *core.ID          `json:"genesisID"`
+	ProfileNonce       *jsonInt          `json:"profileNonce"`
+	AuthClaim          *core.Claim       `json:"claim"`
```

Comment:
I think better call it authClaim, as it is specific claim, and reference that specific auth claim should be provided in this field.